cardano-git-rev/src/Cardano/Git/RevFromGit.hs

```diff
@@ -0,0 +1,32 @@
+module Cardano.Git.RevFromGit (
+      gitRevFromGit
```

Comment:
So we format the export list as follows:
```
module Cardano.Git.RevFromGit 
  ( gitRevFromGit
  ) where
```
You can refer to `Export Lists` in the `STYLE.md`

Reply:
Cheers, I didn't know about the style guide. 👍🏻